src/oncify.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Returns a function that runs only once
+ *
+ * @param {Function}  fn    the function to be made to run only once
+ *
+ * @returns {Function}      the function that runs only once
+ */
+const oncify = function (fn: Function): Function {
+  if (typeof fn !== "function") {
```

Comment:
Since the library is written in typescript, we can skip this check, typescript will scream if the user passes in an argument that isn't a function.

Reply:
Alright. I'll make this change.